test/tekton-tests.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script runs the integration tests with Tekton
+
+# In order to run the tests locally against a standalone Kubernetes cluster
+# and container registry, you must set the $CONTAINER_REGISTRY and $DOCKER_CONFIG_JSON
+# environment variables and login to your registry of choice.
+
+source $(dirname $0)/../vendor/knative.dev/test-infra/scripts/e2e-tests.sh
```

Comment:
You can move `source vendor/.../e2e-tests.sh` into e2e-common.sh and have a single `source` in your *-tests.sh files.